test/service/test_tdr.py

```diff
@@ -342,11 +353,13 @@ def convert_manifest(self, manifest: MutableJSONs, metadata: JSON, bundle_fqid:
                 entry['version'],
                 dss.version_format
             ).strftime(
-                AzulTDRClient.timestamp_format
+                tdr.Plugin.timestamp_format
             )
             if entry['indexed']:
                 entry['size'] = len(json.dumps(metadata[entry['name']]).encode('UTF-8'))
-                entry.update(Checksums.without_values())
+                entry.update(tdr.Checksums.without_values())
+            else:
+                entry['drs_path'] = f"v1_{snapshot_id}_{entry['uuid']}"
```

Comment:
The use of `uuid` here is suprising. I guess it's because this goes from v1 to v2 so therefore you don't have the file_id. That may need a short explanation.

Reply:
Correct, I just wasn't sure what to else to use here. Adding comment.